src/main/java/com/conycomy/MemoryBookCatalog.java

```diff
@@ -0,0 +1,52 @@
+package com.conycomy;
+
+import com.conycomy.dto.Availability;
+import com.conycomy.dto.Book;
+import com.conycomy.dto.BookRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MemoryBookCatalog implements BookCatalog {
+
+	private final List<Book> books;
+
+	public MemoryBookCatalog(List<Book> books) {
+		this.books = books;
+	}
+
+	@Override
+	public List<Availability> checkAvailability(List<BookRequest> requests) {
+		List<Availability> result = new ArrayList<>();
+
+		for (BookRequest bookRequest : requests) {
+			boolean isAvailable = false;
+
+			for (Book book : books) {
+				if (book.title().equalsIgnoreCase(book.title())
+					&& book.author().equalsIgnoreCase(book.author())) {
```

Comment:
```suggestion
			for (Book book : books) {
			if (book.title().equalsIgnoreCase(bookRequest.title()) && book.author().equalsIgnoreCase(bookRequest.author())) {
}
```
현재 book의 title와 book의 title을 비교하고 있는데
bookRequest와 비교해야하는 것이 아닐까 싶습니다!

Reply:
엇 감사합니다!  수정하겠습니다!